common/lib/capa/capa/templates/chemicalequationinput.html

```diff
@@ -2,23 +2,23 @@
 <div id="chemicalequationinput_${id}" class="chemicalequationinput">
     <div class="script_placeholder" data-src="${previewer}"/>
 
-    <div class="${status.classname}" id="status_${id}">
+    <div class="${status.classname}">
 
-    <input type="text" name="input_${id}" id="input_${id}" aria-label="${remove_markup(response_data['label'])}" aria-describedby="answer_${id}" data-input-id="${id}" value="${value|h}"
-   % if size:
-   size="${size}"
-   % endif
-   />
+      <input type="text" name="input_${id}" id="input_${id}" aria-label="${remove_markup(response_data['label'])}"
+             aria-describedby="answer_${id}"
```

Comment:
I personally wouldn't do multiple attributes on the first line and then not do the same on the second line.  Feel free to keep size separate do to conditional.